### PR TITLE
Sl Armor Buff

### PR DESCRIPTION
**Pull request by Alterist**
:cl:
balance: SL armor is now lighter, this means that SLs should now be able to keep up with all members of their squad
/:cl:

I felt like this had to be done, what with the regular marine getting buffs in the form of armor variants

### CODE REVIEW

File: code/modules/clothing/suits/marine_armor.dm [hunk 1]
```diff
@@ -200,6 +199,26 @@ var/list/squad_colors = list(rgb(230,25,25), rgb(255,195,45), rgb(200,100,200),
 	icon_state = "6"
 	armor = list(melee = 30, bullet = 70, laser = 45, energy = 30, bomb = 60, bio = 10, rad = 10)
 
+/obj/item/clothing/suit/storage/marine/M3P/tanker
+	name = "\improper M3 pattern tanker armor"
+	desc = "A modified and refashioned suit of M3 Pattern armor designed to be worn by the loader of a TGMC vehicle crew. While the suit is a bit more encumbering to wear with the crewman uniform, it offers the loader a degree of protection that would otherwise not be enjoyed."
+	icon_state = "tanker"
+
+	New()
```
Comment (DominikPanic) [inline code/modules/clothing/suits/marine_armor.dm]: Relative typepath, turn it into
`/obj/item/clothing/suit/storage/marine/M3P/tanker/New()`
and fix indentation

File: code/modules/clothing/suits/marine_armor.dm [hunk 2]
```diff
@@ -200,6 +199,26 @@ var/list/squad_colors = list(rgb(230,25,25), rgb(255,195,45), rgb(200,100,200),
...
+		..()
+
+/obj/item/clothing/suit/storage/marine/leader
+	name = "\improper B12 pattern leader armor"
+	desc = "A lightweight suit of carbon fiber body armor built for quick movement. Designed in a lovely forest green. Use it to toggle the built-in flashlight."
+	icon_state = "7"
+	armor = list(melee = 50, bullet = 60, laser = 45, energy = 40, bomb = 40, bio = 15, rad = 15)
+	slowdown = SLOWDOWN_ARMOR_LIGHT
+
+	New(loc,expected_type 	= type,
```
Comment (DominikPanic) [inline code/modules/clothing/suits/marine_armor.dm]: Same here.

File: code/modules/clothing/suits/marine_armor.dm [hunk 3]
```diff
@@ -200,6 +199,26 @@ var/list/squad_colors = list(rgb(230,25,25), rgb(255,195,45), rgb(200,100,200),
...
+		new_name[] 		= list(MAP_ICE_COLONY = "\improper B12 pattern leader snow armor"))
+		..(loc,expected_type,new_name)
```
Comment (DominikPanic) [inline code/modules/clothing/suits/marine_armor.dm]:
```suggestion
	return ..()
```
All arguments are passed by default.

File: code/modules/clothing/suits/marine_armor.dm [hunk 4]
```diff
@@ -200,6 +199,26 @@ var/list/squad_colors = list(rgb(230,25,25), rgb(255,195,45), rgb(200,100,200),
...
 	armor = list(melee = 30, bullet = 70, laser = 45, energy = 30, bomb = 60, bio = 10, rad = 10)
 
+/obj/item/clothing/suit/storage/marine/M3P/tanker
+	name = "\improper M3 pattern tanker armor"
+	desc = "A modified and refashioned suit of M3 Pattern armor designed to be worn by the loader of a TGMC vehicle crew. While the suit is a bit more encumbering to wear with the crewman uniform, it offers the loader a degree of protection that would otherwise not be enjoyed."
+	icon_state = "tanker"
+
+	New()
+		select_gamemode_skin(type)
+		..()
```
Comment (DominikPanic) [inline code/modules/clothing/suits/marine_armor.dm]:
```suggestion
	return ..()
```

File: code/modules/clothing/suits/marine_armor.dm [hunk 5]
```diff
@@ -626,9 +618,9 @@ var/list/squad_colors = list(rgb(230,25,25), rgb(255,195,45), rgb(200,100,200),
 	armor = list(melee = 65, bullet = 80, laser = 40, energy = 25, bomb = 35, bio = 10, rad = 10)
 	slowdown = SLOWDOWN_ARMOR_LIGHT
 
-	New()
-		select_gamemode_skin(type)
-		..()
+/obj/item/clothing/suit/storage/marine/M3S/New()
+	select_gamemode_skin(type)
+	..()
```
Comment (DominikPanic) [inline code/modules/clothing/suits/marine_armor.dm]:
```suggestion
	return ..()
```

File: code/modules/clothing/suits/marine_armor.dm [hunk 6]
```diff
@@ -615,9 +607,9 @@ var/list/squad_colors = list(rgb(230,25,25), rgb(255,195,45), rgb(200,100,200),
 	slowdown = SLOWDOWN_ARMOR_LIGHT
 	allowed = list(/obj/item/weapon/gun/launcher/rocket)
 
-	New()
-		select_gamemode_skin(type)
-		..()
+/obj/item/clothing/suit/storage/marine/M3T/New()
+	select_gamemode_skin(type)
+	..()
```
Comment (DominikPanic) [inline code/modules/clothing/suits/marine_armor.dm]:
```suggestion
	return ..()
```

File: code/modules/clothing/suits/marine_armor.dm [hunk 7]
```diff
@@ -641,9 +633,9 @@ var/list/squad_colors = list(rgb(230,25,25), rgb(255,195,45), rgb(200,100,200),
 	flags_cold_protection = CHEST|GROIN|ARMS|LEGS|FEET
 	flags_heat_protection = CHEST|GROIN|ARMS|LEGS|FEET
 
-	New()
-		select_gamemode_skin(type)
-		..()
+/obj/item/clothing/suit/storage/marine/M35/New()
+	select_gamemode_skin(type)
+	..()
```
Comment (DominikPanic) [inline code/modules/clothing/suits/marine_armor.dm]:
```suggestion
	return ..()
```

File: code/modules/clothing/suits/marine_armor.dm [hunk 8]
```diff
@@ -51,8 +48,8 @@ var/list/squad_colors = list(rgb(230,25,25), rgb(255,195,45), rgb(200,100,200),
 		/obj/item/storage/bible,
 		/obj/item/storage/sparepouch,
 		/obj/item/storage/large_holster/machete,
-		/obj/item/storage/belt/gun/m4a3,
-		/obj/item/storage/belt/gun/m44,)
+		/obj/item/weapon/claymore/,
```
Comment (spookydonut) [inline code/modules/clothing/suits/marine_armor.dm]:
```suggestion
		/obj/item/weapon/claymore,
```

File: code/modules/clothing/suits/marine_armor.dm [hunk 9]
```diff
@@ -51,8 +48,8 @@ var/list/squad_colors = list(rgb(230,25,25), rgb(255,195,45), rgb(200,100,200),
...
+		/obj/item/storage/belt/gun/)
```
Comment (spookydonut) [inline code/modules/clothing/suits/marine_armor.dm]:
```suggestion
		/obj/item/storage/belt/gun)
```

**Comment by Alterist** [comment]
Conflicts resolved, suggestions added

**Pull request merged**